packages/gatsby-telemetry/package.json

```diff
@@ -50,9 +50,10 @@
   },
   "scripts": {
     "build": "babel src --out-dir lib --ignore \"**/__tests__\",\"**/__mocks__\" --extensions \".ts,.js\"",
-    "prepare": "cross-env NODE_ENV=production npm run build",
+    "prepare": "cross-env NODE_ENV=production npm run build && npm run typegen",
     "jest": "jest",
     "postinstall": "node src/postinstall.js || true",
+    "typegen": "rimraf \"lib/**/*.d.ts\" && tsc --emitDeclarationOnly --declaration --declarationDir lib/",
```

Comment:
Can you install rimraf as a devDep please

Reply:
Done